languageServer/src/DvcLanguageServer.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  TextDocuments,
+  InitializeResult,
+  ServerCapabilities,
+  _Connection,
+  TextDocumentPositionParams,
+  CodeActionParams
+} from 'vscode-languageserver/node'
+import { TextDocument } from 'vscode-languageserver-textdocument'
+import { DvcTextDocument } from './DvcTextDocument'
+
+const documentSelector = [
+  {
+    language: 'yaml'
+  },
+  {
+    pattern: '**/*.{dvc,dvc.lock}'
+  },
+  {
+    language: 'json'
+  },
+  {
+    language: 'toml'
+  },
+  {
+    language: 'python'
+  }
+]
+export class DvcLanguageServer {
```

Comment:
I'm think about removing the `Dvc*` prefix from these names. It seems redundant and three more characters to type